jetbrains-core/resources/META-INF/ext-java.xml

```diff
@@ -9,7 +9,10 @@
     <extensions defaultExtensionNs="aws.toolkit.lambda">
         <runtimeGroup implementation="software.aws.toolkits.jetbrains.services.lambda.java.JavaRuntimeGroup"/>
         <builder id="JAVA" implementationClass="software.aws.toolkits.jetbrains.services.lambda.java.JavaLambdaBuilder"/>
-        <sam.debugSupport id="JAVA" implementationClass="software.aws.toolkits.jetbrains.services.lambda.java.JavaSamDebugSupport"/>
+        <sam.runtimeDebugSupport id="JAVA" implementationClass="software.aws.toolkits.jetbrains.services.lambda.java.JavaRuntimeDebugSupport"/>
```

Comment:
should we split the runtime variants as well?

Reply:
I was thinking that, as well as maybe pushing id into the implementations themselves. Either way it's a bit weird, but explicit registering seems to make the most sense. Let me see if that would make sense to put in this PR.